cloudferrylib/os/image/glance_image.py

```diff
@@ -68,8 +68,10 @@ def get_db_connection(self):
     def get_client(self):
         """ Getting glance client """
 
-        endpoint_glance = self.identity_client.get_endpoint_by_service_name(
```

Comment:
Looks like `get_endpoint_by_service_name` method is not used anywhere else, lets remove it